src/screens/QuestionScreen/QuestionScreen.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity, SafeAreaView, TextInput, Alert, Linking } from 'react-native';
+import { firebase } from '../../config/config';
+import { LOGO } from '../../config/styles.js';
+import styles from './styles';
+import moment from 'moment';
+import Icon from 'react-native-vector-icons/Ionicons';
+import ImagePicker from 'react-native-image-crop-picker';
+import * as Progress from 'react-native-progress';
+
+class QuestionScreen extends  React.Component {
+    state = {queryInput : '', setImage : null, setDisplayImage: null, transferred: 0};
+
+    takePhotoFromLib = () => {
+        const {setImage} = this.state
+        this.setState({transferred: 0})
+        ImagePicker.openPicker({
+          width: 300,
+          height: 300,
```

Comment:
As I said before, hope these will not affect to different devices(better to check with some tablets also since they are also listed in mobile devices category)

Reply:
I'll remove the cropping option from this picker. This height and width are only valid when cropping of an image is enabled. Removing cropping makes height and width variable according to the device.

Reply:
I'm not referring to this specific scenario. I'm talking about the whole styles that are defined without device height/width variables.

Reply:
Updated Dimensions